pages/apim/3.x/installation-guide/configuration/repositories/installation-guide-repositories-jdbc.adoc

```diff
@@ -36,7 +36,7 @@ This repository plugin is for connecting to SQL databases.
 
 == Install the JDBC plugin
 
-NOTE: Because the plugin is part of the default distribution from APIM version 3.5, you only need to complete these steps for version 3.4 or earlier.
```

Comment:
@jgiovaresco - could you review this change?

Reply:
The plugin is included by default on all supported version so I think we can simplify the doc and focus only on the driver part (needed in all versions as drivers aren't provided)

Reply:
I have updated the documents.